.github/CODEOWNERS

```diff
@@ -84,6 +84,7 @@
 /src/stash-debug                    @MaterializeInc/surfaces
 /src/storage                        @MaterializeInc/storage
 /src/storage-client                 @MaterializeInc/storage
+/src/test-macro                     @MaterializeInc/qa
```

Comment:
@philip-stoev I hope this is ok, otherwise can also assign it to myself directly.

Reply:
Yes, it is ok, thanks.